app/services/validation_beta_service.rb

```diff
@@ -30,6 +30,15 @@ def tell_induction_coordinators_to_check_ect_and_mentor_information
     end
   end
 
+  def tell_ects_and_mentors_to_add_validation_information
+    ParticipantProfile::ECF
+      .includes(:ecf_participant_eligibility, :ecf_participant_validation_data, :school_cohort, :school)
+      .where(school_cohort: { cohort: Cohort.current }) # Chosen program
+      .where(ecf_participant_eligibility: { participant_profile_id: nil })
+      .or(ParticipantProfile::ECF.where(ecf_participant_validation_data: { participant_profile_id: nil }))
```

Comment:
That seems a bit weird - why do we say 'where the participant eligibility has no profile', rather than 'where there is no participant eligibility? 

```
      .where(ecf_participant_eligibility: nil)
      .or(ParticipantProfile::ECF.where(ecf_participant_validation_data: nil))
```

Reply:
These are both associations and we want to check that there are either zero `ecf_participant_eligibility` or zero `ecf_participant_validation_data` associated.

Reply:
Yeah, that part makes sense. What feels weird is 
```
.where(ecf_participant_eligibility: { participant_profile_id: nil })
```
vs
```
.where(ecf_participant_eligibility: nil)
```
Are we not be able to do the latter because `ecf_participant_eligibility` is a `has_one` association?

Reply:
Ah, yes, will return to it. I thought it was a `has_many`.

Reply:
@kicferk1 I tried this in the console and I don't think that works.

Reply:
Ah well, too bad. We have tests that look convincing, so it should behave fine :)